htdocs/compta/bank/class/api_bankaccounts.class.php

```diff
@@ -305,16 +305,16 @@ public function transfer($bankaccount_from_id = 0, $bankaccount_to_id = 0, $date
 			);
 		} else {
 			$this->db->rollback();
-			throw new RestException(500, $accountfrom->error.' '.$accountto->error);
+			throw new RestException(500, $accountfrom->error . ' ' . $accountto->error);
 		}
 	}
 
 	/**
 	 * Update account
 	 *
-	 * @param 	int    $id              ID of account
-	 * @param 	array  $request_data    data
-	 * @return	Object              	Object with cleaned properties
+	 * @param int    $id              ID of account
+	 * @param array  $request_data    data
+	 * @return int
```

Comment:
Why replacing the Object with int ?

Reply:
You're right it's a mistake. Iirc it's because I first did this code for the v14 or 15 when the endpoints were returning int. 
```suggestion
	 * @return Object
```